src/libraries/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -470,6 +470,7 @@ public static void ParseJson_UnseekableStream_BadBOM(string json)
                     new WrappedMemoryStream(canRead: true, canWrite: false, canSeek: false, data)));
         }
 
+        [SkipOnCoreClr("Long running test on Checked mode", RuntimeConfiguration.Checked)]
```

Comment:
```suggestion
        [SkipOnCoreClr("https://github.com/dotnet/runtime/issues/45464", RuntimeConfiguration.Checked)]
```

I think we should point to the issue and leave it open since this is only a temporary fix (i.e we eventually want to see if we can make them faster without losing test coverage). Applies here and elsewhere

Reply:
Sounds good. I thought we didn't want to enable them for Checked CoreCLR. I will do that instead.